cartridge/roles/metrics.lua

```diff
@@ -1,5 +1,6 @@
 local cartridge = require('cartridge')
 local argparse = require('cartridge.argparse')
+local hotreload_supported, hotreload = pcall(require, 'cartridge.hotreload')
```

Comment:
It's one of the points covered by this patch, yes.

It could be achieved by simply replacing `package.loaded` with `pcall(require)`, but I think such approach isn't good since being able to require `cartridge.hotreload` doesn't always mean that we are working with cartridge application (even though in most cases it is). So I moved it to a role, similar to `crud`, since using role means that you are surely working with cartridge app, and if you work with cartridge app and expect cartridge features (like hotreload) to work, it is expected that you'll use metrics cartridge role.